agent/iptables_helper.py

```diff
@@ -13,6 +12,58 @@
 OUTPUT_CHAIN = 'WOTT_OUTPUT'
 INPUT_CHAIN = 'WOTT_INPUT'
 
+# Fix a bug in python-iptables by monkey-patching a couple of methods.
+# The bug is actually fixed in:
+#  https://github.com/chruss2/python-iptables/commit/282c790738a111b1ddc27b43ecb0acfab8b09024
+#  and the bugfix is gonna be released in the next (after 0.14.0) release of python-iptables.
+if iptc.version.__version__ <= '0.14.0':
```

Comment:
This will be True even if `_version__` is something like `0.14.0~ds-1` like in [Debian bullseye](https://packages.debian.org/bullseye/python3-iptables) where the bug is **not** fixed.

Reply:
Pls don't mix OS package version with python package version.

Reply:
Even so, if we have version like `0.14.0-dev` it will be considered "bug-free" even though it's not.

Reply:
https://github.com/ldx/python-iptables/blob/master/iptc/version.py#L4
https://github.com/ldx/python-iptables/commit/e42e691732ec93d1bc05a954012246e78827f8d3#diff-7e58f41d9cf532f0e3a8f83679921b18R4

Reply:
Fixed in my last commit